configuration/src/main/kotlin/com/malinskiy/marathon/config/Configuration.kt

```diff
@@ -142,37 +143,37 @@ data class Configuration private constructor(
         return result
     }
      data class Builder(
-        val name: String,
-        val outputDir: File,
-        var analyticsConfiguration: AnalyticsConfiguration = AnalyticsConfiguration.DisabledAnalytics,
-        var poolingStrategy: PoolingStrategyConfiguration = PoolingStrategyConfiguration.OmniPoolingStrategyConfiguration,
-        var shardingStrategy: ShardingStrategyConfiguration = ShardingStrategyConfiguration.ParallelShardingStrategyConfiguration,
-        var sortingStrategy: SortingStrategyConfiguration = SortingStrategyConfiguration.NoSortingStrategyConfiguration,
-        var batchingStrategy: BatchingStrategyConfiguration = BatchingStrategyConfiguration.IsolateBatchingStrategyConfiguration,
-        var flakinessStrategy: FlakinessStrategyConfiguration = FlakinessStrategyConfiguration.IgnoreFlakinessStrategyConfiguration,
-        var retryStrategy: RetryStrategyConfiguration = RetryStrategyConfiguration.NoRetryStrategyConfiguration,
-        var filteringConfiguration: FilteringConfiguration = FilteringConfiguration(emptyList(), emptyList()),
-
-        var ignoreFailures: Boolean = false,
-        var isCodeCoverageEnabled: Boolean = false,
-        var strictMode: Boolean = false,
-        var uncompletedTestRetryQuota: Int = Integer.MAX_VALUE,
-
-        var testClassRegexes: Collection<Regex> = listOf(Regex("^((?!Abstract).)*Test[s]*$")),
-        var includeSerialRegexes: Collection<Regex> = emptyList(),
-        var excludeSerialRegexes: Collection<Regex> = emptyList(),
-
-        var testBatchTimeoutMillis: Long = DEFAULT_BATCH_EXECUTION_TIMEOUT_MILLIS,
-        var testOutputTimeoutMillis: Long = DEFAULT_OUTPUT_TIMEOUT_MILLIS,
-        var debug: Boolean = true,
-
-        var screenRecordingPolicy: ScreenRecordingPolicy = ScreenRecordingPolicy.ON_FAILURE,
-
-        var analyticsTracking: Boolean = false,
-        var deviceInitializationTimeoutMillis: Long = DEFAULT_DEVICE_INITIALIZATION_TIMEOUT_MILLIS,
-
-        var outputConfiguration: OutputConfiguration = OutputConfiguration(),
-        var vendorConfiguration: VendorConfiguration = VendorConfiguration.EmptyVendorConfiguration(),
+         val name: String,
+         val outputDir: File,
```

Comment:
Not everything is reformatted according to the embedded code style. It works as an incremental code style formatter - you change something in a file and once you commit it will have approriate format

Reply:
Added a newline